web/src/components/Plugin/UI/plugin.tsx

```diff
@@ -38,6 +39,8 @@ export const PluginForm: React.FC<Props> = ({ name, renderForm, form }) => {
   switch (name) {
     case 'basic-auth':
       return <BasicAuth form={form} />
+    case 'proxy-mirror':
+      return <ProxyMirror form={form} />
```

Comment:
why we need this switch？

Reply:
This Form receives Plug-in Name and returns the target Component, use Switch-Case is ok here.